test/acceptance/features/steps/persistent_clusters.py

```diff
@@ -105,67 +107,48 @@ def cleanup_controller_namespace(self, namespace: str):
         # they are.
         resource_names = ["registeredservices", "servicecatalogs", "serviceclaims", "serviceclasses", "clusterenvironments"]
         resources = list(map(lambda s: f"{s}.primaza.io", resource_names))
-        for crd in resources:
-            names, _ = cmd.run(f"kubectl get {crd} -o name -n {namespace}")
-            for resource in names.splitlines():
-                patch = '{"metadata": {"finalizers": []}}'
-                cmd.run(f'kubectl delete -n {namespace} {resource} --force --timeout=60s')
-                cmd.run(f'kubectl patch -n {namespace} {resource} -p \'{patch}\' --type=merge')
-        x = ",".join(resources)
-        out, err = cmd.run(f"kubectl delete -n {namespace} {x} --all --ignore-not-found")
-        assert err == 0, "failed to run command!"
-
-        # delete any deployments in the namespace
-        cmd.run(f"kubectl delete -n {namespace} deployments --all --ignore-not-found --timeout=30s")
-        names, _ = cmd.run(f"kubectl get deployments.apps -o name -n {namespace}")
-        for resource in names.splitlines():
-            patch = '{"metadata": {"finalizers": []}}'
-            cmd.run(f'kubectl patch -n {namespace} {resource} -p \'{patch}\' --type=merge')
+        resources.append("deployments")
```

Comment:
This and the other places we add `deployments` should have its group name added, e.g. "deployments.apps".

Reply:
thanks, great catch

Reply:
done